src/pages/Login/Signup.js

```diff
@@ -85,6 +111,13 @@ class Signup extends Component {
             );
         }
 
+        let EmailValidError = '';
+        if(this.state.emailAlreadyExists) {
+            EmailValidError =  (
+                <ErrorMessage content="This email already exists" />
```

Comment:
I think a better error message would be This email address is already in use. What do you think?

Reply:
I agree. I've committed the change.